ext/node/polyfills/tty.js

```diff
@@ -17,7 +20,7 @@ function isatty(fd) {
     return false;
   }
   try {
-    return Deno.isatty(fd);
+    return op_is_terminal(fd);
```

Comment:
Let's add a note here that says:

> TODO: Treat `fd` as real file descriptors. Currently, rid 0, 1, 3 correspond to fd 0, 1, 2 (stdin, stdout, stderr). This may change in the future.

Reply:
Do you mean "rid 0, 1, 2 correspond to fd 0, 1, 2"?

Reply:
Er, so that API will probably be broken once we land it. Should we add a special op just for Node compat?

Reply:
I wonder if we should. It just needs to be something like `ResourceHandle::Fd(fd).is_terminal()`, I think.